toolbox/bin/variables.sh

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env sh
 set -eu -o pipefail
 
+echo "Rackspace Toolbox - 1.7.1"
```

Comment:
Is there a way to _not_ hardcode this? I guess that is why you wrote that other test, to prevent us from accidentally shipping this without changing it?

Reply:
I spent some time trying to figure out a way. We build the image from the `./toolbox` directory. The changelog file is not there. We'd have to copy it first, and then extract it from the script. Maybe we should do that, but I just started to think it was an overkill.

Reply:
Eh, yeah. I could see that. I don't feel strongly about it, so long as we're protected against shipping this with a mismatched number.

Reply:
We could also release new versions, see what the image ref is, and include that in the release. So we could match container ids with releases, without hard coding the release. It adds a lookup step, but at least it won't be hard-coded.